### ISSUE TITLE
pyangbindJSON quoting values (ReferencePathType)

### ISSUE THREAD

**Issue by mike-albano** [robshakir/pyangbind#139]
The leaf-list key is quoted even though it's an uint8. The leaf node is not quoted.

Example:
{
  "key1": {
    "1": {
      "config": {
        "id": 1 << not quoted.
        },
        "id": "1" << quoted.
      },

Corrosponding model:
  grouping radio-common-config {
    description
      "Configuration items common to all Radio interfaces, independent
      of frequency";

    leaf id {
      type uint8;
      description
        "Unique ID of the radio.";
    }

container radios {
      description
        "Top level container for radios, including configuration
        and state data.";

      list radio {
        key "id";
        description
          "The list of radios on the device.";

        leaf id {
          type leafref {
            path "../config/id";
          }

**Comment by nkaliape** [comment]
Yes. Same issue hit in Openconfig VLAN as well + for leaf-ref also.
